.github/workflows/release.yaml

```diff
@@ -0,0 +1,60 @@
+name: Release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  build:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup kustomize
+        uses: imranismail/setup-kustomize@v2
+
+      - name: Generate release manifests
+        run: |
+          make manifests
+          kustomize build ./manifests > ./manifests/vm-console-proxy.yaml
```

Comment:
@akrejcir Do you prefer to have a new make target? For example:
```
.PHONY: kustomize
kustomize: manifests
	kustomize build ${MANIFETS_PATH} > ./manifests/${PROJECT_NAME}-${RELEASE_VERSION}.yaml
```

Reply:
A `make` target is a good idea, but without the `${RELEASE_VERSION}` in the filename. And it can be generated into an `_out` directory, or some similar name, which will be ignored in `.gitignore`.

Reply:
Sorry, why it should be generated into `_out` directory? Why we need to ignore it? I mean it won't be committed here, just be uploaded to GitHub release.

Reply:
A developer can run the make command on their machine and it would generate the file, which should be ignored, so it is not committed by mistake.